serve/db/db.go

```diff
@@ -94,7 +116,13 @@ func (db *DB) Transact(f func() (commit bool)) (bool, error) {
 
 type Params map[string]interface{}
 
+type ExecFunc func(sql string, args Params) (*rdsdataservice.ExecuteStatementOutput, error)
+
 func (db *DB) Exec(sql string, args Params) (*rdsdataservice.ExecuteStatementOutput, error) {
+	return db.ExecInTransaction(nil, sql, args)
+}
+
+func (db *DB) ExecInTransaction(transactionID *string, sql string, args Params) (*rdsdataservice.ExecuteStatementOutput, error) {
```

Comment:
[taste] It doesn't feel idiomatic to me to pass a pointer to string as an input param. I think it would be more natural looking to pass a string and use empty string to mean create-a-transaction.

Reply:
yeah the sig is a little odd. i'm hesitant to make it work like that tho bc we don't have use for it and we already have a way to execute in a tx. i made it a regular string and tweaked the name a bit, which i think is a little better.